build.groovy

```diff
@@ -63,29 +58,53 @@ node("${env.OS.toLowerCase()}") {
     }
 
     stage("Create environment") {
+        
         env.STAGE_STATUS = "Creating conda environment"
-        sh '''
+        
+        if (env.OS.toLowerCase() == "mac") {
+          condaPath = "/tmp/" + sh(script: '{ date "+%H:%M:%S:%m"; echo $WORKSPACE; } | md5 | tr -d "\n";', returnStdout: true) 
+
+          sh """
+            curl -o miniconda.sh  https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+            bash miniconda.sh -b -p ${condaPath}
+            """
+        } else {
+          condaPath = "/home/jenkins/.conda/"
+        } 
+        
+        println(condaPath)
```

Comment:
It would be nice if this also printed something like "Path to Anaconda installation..." so that people know what this debug output is without looking back here.